src/Autocomplete/assets/dist/controller.js

```diff
@@ -227,7 +227,11 @@ _default_1_instances = new WeakSet(), _default_1_getCommonConfig = function _def
     const plugins = {};
     const isMultiple = !this.selectElement || this.selectElement.multiple;
     if (!this.formElement.disabled && !isMultiple) {
-        plugins.clear_button = { title: '' };
+        const placeholderOptions = Array.from(this.selectElement.options)
+            .filter((option) => option.value === '');
```

Comment:
Maybe use `Array.from(...).find()` here ?
(especially as the placeholder will be first element in most of the cases)

Reply:
Even `.some()` that will return you the boolean you need L232 ?

Reply:
(yeah.. in the .ts file 😅 )